jumpscale/clients/gedis/gedis.py

```diff
@@ -90,8 +92,19 @@ def __init__(self):
         self._loaded_actors = {}
         self._loaded_modules = []
         self.actors = None
+        self.identity = j.core.identity
+        self._auth_server()
         self._load_actors()
 
+    def _auth_server(self):
+        server_public_key = binascii.unhexlify(self.execute("core", "get_pubkey", die=True).result)
```

Comment:
when getting the pubkey of the server like this we expose ourselves for MITM attack